tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -22,6 +22,12 @@ select table_name from create_hypertable('conditions', 'timec');
 
 -- schema tests
 \c :TEST_DBNAME :ROLE_SUPERUSER
+SET client_min_messages = ERROR;
+DROP TABLESPACE IF EXISTS tablespace1;
+DROP TABLESPACE IF EXISTS tablespace2;
+SET client_min_messages = NOTICE;
```

Comment:
this shouldnt be needed, if this is needed this means tests are not properly classified as solo tests or not properly restoring state after they are done

Reply:
Removed.